kernel/src/process.rs

```diff
@@ -272,6 +273,15 @@ pub enum StoppedExecutingReason {
     KernelPreemption,
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub struct BinaryVersion(NonZeroU32);
```

Comment:
Please document this type. A simple sentence describing it's purpose or meaning would be sufficient.

Reply:
Not quite a sentence, but added a very brief description. I think it suffices.